src/_gettsim/social_insurance_contributions/ges_pflegev.py

```diff
@@ -322,26 +328,28 @@ def ges_pflegev_beitr_selbst_m_ohne_zusatz_fuer_kinderlose(
     Parameters
     ----------
 
-    _ges_krankenv_bemessungsgrundlage_eink_selbst
-        See :func:`_ges_krankenv_bemessungsgrundlage_eink_selbst`.
+    _ges_krankenv_bemessungsgrundlage_eink_selbständig
+        See :func:`_ges_krankenv_bemessungsgrundlage_eink_selbständig`.
 
-    ges_pflegev_beitr_satz
-        See :func:`ges_pflegev_beitr_satz`.
+    ges_pflegev_beitr_satz_arbeitnehmer
+        See :func:`ges_pflegev_beitr_satz_arbeitnehmer`.
 
     Returns
     -------
     Monthly care insurance contributions for self employed income.
```

Comment:
Harmonise with other docstrings. Always "long term care", monthly can be dropped.